### DISCUSSION TITLE
Improved File Transfer log

### DISCUSSION THREAD

**Discussion by BobWilkes** [rustdesk/rustdesk#5367] [Feature Request]
PLS consider revising the log list:
1. Give me an option to NOT display individual files after they copy OK: "Display only failed copy"
    In that case, display a summary at the end: "NN files copied OK"
3. Modify layout to reduce vertical space.
    Current layout shows only 7 files.
    Pls reformat to one line  per file: File name, Size, Status.  If copied OK, a green check mark; If not a Red X.
   Should then be able to list ~30 files

**Comment by rustdesk** [comment]
Yes, we do need to improve this. 
@21pages we have discussed (get it from our chat history) before, let's follow our discussion first.

**Reply by BobWilkes** [reply]
Didn't know about that discussion. AND I couldn't find any entries related to File Transfer Log...
NOTE TO MODERATORS: Feel free to move this to that discussion, if appropriate.

**Reply by rustdesk** [reply]
That's our internal team discussion.

**Reply by BobWilkes** [reply]
While I'm here:
What I was thinking about is:
1. Add option to "Display log summary"
2. Change log entries to FileOne.abc <% complete while transferring> Then FileOne.abc copy AOK <Green check>
    If the 'Display log summary' option is selected, this entry is removed from the list.
    FileTwo.def ... COPY FAILED <Red X>
   @End of job  either ALL of 92 files copied <Green check>
                           or  91 of 92 files copied <Red X>
